src/main/java/org/opentripplanner/standalone/OTPMain.java

```diff
@@ -64,6 +70,18 @@ public static void main(String[] args) {
                 jc.usage();
                 System.exit(0);
             }
+
+            if (params.version) {
+                // Disable logging, we don't want to see anything else than the intended output below.
+                ch.qos.logback.classic.Logger mavenVersionLogger =
```

Comment:
I'll take it out. Coudn't figure out any other way of disabling log output for this without affecting the default.